package-lock.json

```diff
@@ -4443,7 +4456,7 @@
       "requires": {
         "async": "^2.1.5",
         "find-process": "^1.0.5",
-        "fs-ext": "github:baudehlo/node-fs-ext#7c9824f3dc330e795aa13359d96252860bd3a684",
+        "fs-ext": "github:baudehlo/node-fs-ext#2ba366d9fc67ef3ab165e239068924b276ecf249",
```

Comment:
Why is this needed?

Reply:
Hmm... Seems completely redundant. Must have been updated when I ran npm install.

Ill remove it from the commit.

Reply:
oh, wait I thought it was in `package.json` but it's in the package lock. NVM then!